plugins/experimental/remap_stats/remap_stats.cc

```diff
@@ -243,15 +254,15 @@ TSPluginInit(int argc, const char *argv[])
     TSDebug(DEBUG_TAG, "Plugin registration succeeded");
   }
 
-  auto config                 = new config_t;
+  config_t *const config      = new config_t;
```

Comment:
This is considered bad style, because you're repeating the type name. Using `auto` in this case is better.

Reply:
restores this back to auto